migrations/releases/6.0.0/config/backend/create/ml_nvsk_improvement_projects_report_table_sl.json

```diff
@@ -0,0 +1,382 @@
+{
+    "request": {
+      "reportId": "ml_nvsk_improvement_projects_report_table_sl",
+      "config": {
+        "streamQuery": true,
+        "reportConfig": {
+          "id": "ml_nvsk_improvement_projects_report_table_sl",
+          "mergeConfig": {
+            "postContainer": "reports",
+            "container": "reports",
```

Comment:
both should be variables, kindly check any existing report confing from this folder.